src/deluge/io/midi/midi_engine.cpp

```diff
@@ -911,7 +917,7 @@ bool lastWasNoteOn = false;
 #endif
 
 void MidiEngine::midiMessageReceived(MIDIDevice* fromDevice, uint8_t statusType, uint8_t channel, uint8_t data1,
-                                     uint8_t data2, uint32_t* timer) {
+                                     uint8_t data2, uint32_t* timer, bool notDoingLoopback) {
```

Comment:
This isn't necessary, the check can be on received device != loopback device

Reply:
Thanks for pointing that out, felt clumsy while typing it in; now cleaner